### PR TITLE
Update Schlitzohr's Oakwood

### PR DESCRIPTION
**Pull request by loicreynier**

### CODE REVIEW

**Review by sibir-ine — CHANGES_REQUESTED** [review]
@loicreynier, the mod still has some wild edits, namely the records **00009B96**, **00009B97**, & **00009BB9**. If you want, you can remove the cleaning data instead & update the `wildEdits` message, or we can do it in a separate PR.

**Comment by loicreynier** [comment]
I removed the cleaning data and added this as `wildEdits`, is it what you expected?

File: masterlist.yaml [hunk 1]
```diff
@@ -19368,7 +19368,7 @@ plugins:
     msg:
       - <<: *wildEdits
         subs: [ 'Remove **00009B95**, **00009B96**, **00009B97**, & **00009BB9** from **AnotherOakwood.esp**.' ]
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        subs: [ 'Remove **00009B96**, **00009B97**, & **00009BB9** from **AnotherOakwood.esp**.' ]
```

File: masterlist.yaml [hunk 2]
```diff
@@ -19368,7 +19368,7 @@ plugins:
...
-        condition: 'checksum("AnotherOakwood.esp", EE69FD67) or checksum("AnotherOakwood.esp", BB723686)'
+        condition: 'checksum("AnotherOakwood.esp", 1B9CB7E5) or checksum("AnotherOakwood.esp", EE69FD67) or checksum("AnotherOakwood.esp", BB723686)'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        condition: 'checksum("AnotherOakwood.esp", 1B9CB7E5)'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]
Yes, the message just needs to be edited since one of those records is no longer a wild edit. The old checkums & cleaning data (I can't add a suggestion for this) can be dropped too.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**